extension/basicauth/README.md

```diff
@@ -0,0 +1,42 @@
+# Basic Authenticator
+
+This extension implements `configauth.ServerAuthenticator` to authenticate clients using HTTP Basic Authentication. The authenticator type has to be set to `basicauth`.
```

Comment:
Please, state that this authenticator provides the attributes `subject` and `raw` to the client info in the auth data. This is missing from most (all?) current authenticators, but we should add it.

Reply:
I added the section to the readme file, but probably need help with phrasing.